tests/test_indices.py

```diff
@@ -702,10 +702,10 @@ def test_standardized_precipitation_evapotranspiration_index(
             tas = tasmax - 2.5
             tasmin = tasmax - 5
             wb = xci.water_budget(pr, None, tasmin, tasmax, tas)
-            wb = wb + convert_units_to("1 mm/d", wb, context="hydro")
-        fitkwargs = {}
         if method == "APP":
-            fitkwargs["floc"] = 0
+            # same offset as in climate indices
+            offset = convert_units_to("1 mm/d", wb, context="hydro")
```

Comment:
```suggestion
            offset = convert_units_to("1000 mm/month", wb, context="hydro")
```
Isn't this the hardcoded offset of `climate_indices` ?
https://github.com/monocongo/climate_indices/blob/d108eee982abae06a415e888319b8078af868558/src/climate_indices/indices.py#L280C62-L280C69

Reply:
yes, climate_indices and xclim do match, but with the current datasets used in our test, we stil have negative values with the climate_indices offset. I wll wait for a next PR where we show many comparisons to many other libraries to sort this out

Reply:
but I agree, I didn't have the correct offset here